pkg/testcase/restart.go

```diff
@@ -0,0 +1,21 @@
+package testcase
+
+import (
+	"github.com/rancher/distros-test-framework/factory"
+	"github.com/rancher/distros-test-framework/shared"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestRestartService() {
+	product, err := shared.GetProduct()
+	Expect(err).NotTo(HaveOccurred(), "failed to get product")
+	c := factory.ClusterConfig(GinkgoT())
+
+	_, err = shared.ManageService(product, "restart", "server", c.ServerIPs)
+	Expect(err).NotTo(HaveOccurred(), "failed to get systemctl command")
+
+	_, err = shared.ManageService(product, "restart", "agent", c.AgentIPs)
+	Expect(err).NotTo(HaveOccurred(), "failed to get systemctl command")
```

Comment:
https://github.com/rancher/distros-test-framework/pull/50/files#r1414613080

Reply:
message here should be - restart service failed for agent with ip: %s

Reply:
sure